DoctorWho2/Controllers/EpisodeController.cs

```diff
@@ -35,6 +38,18 @@ public async Task<ActionResult<IEnumerable<EpisodeDto>>> GetEpisodes()
             return Ok(EpisodeDtoList);
 
         }
+        [HttpPost]
+        public async Task<ActionResult<int>> CreateEpisode(EpisodeForCreateDto episode)
+        {
+            EpisodeForCreateDtoValidator validator = new EpisodeForCreateDtoValidator();
+
+            ValidationResult results = validator.Validate(episode);
+            var EpisodeEntity = _mapper.Map<Episode>(episode);
+            await _episodeRepository.AddEpisodeAsync(EpisodeEntity);
+            await _episodeRepository.SaveChanges();
+
```

Comment:
I used episodeRepository immediatlly , didn't create unitOfWork and service as in previous req , since I feel by repo its more direct .